ops/lib.rs

```diff
@@ -308,7 +314,14 @@ fn codegen_sync_ret(
     }
   };
 
+  let result_wrapper = match is_result(&**ret_type) {
```

Comment:
While this does allow ops to avoid returning `Ok(ret)` in favour of just `ret`, it seems to me like this leaves a lot of dead code lying around.

If I'm not badly mistaken, this should be enough for a non-result type:
```rs
if !is_result(&**ret_type) {
  if is_unit(&**ref_type) {
    return quote! {};
  }
  return ok_block;
}
```
Might be that some customisation is required, but basically that should be enough I think. In the actual generated Rust it'd then be something like this:

```rs
fn op_void_decl(scope, args, rv) {
  let ret: () = op_void();
  // Nothing to do
}

fn op_not_void_decl(scope, args, rv) {
  let not_void: bool = op_not_void();
  match deno_core::serde_v8::to_v8(scope, v) {
    Ok(ret) => rv.set(ret),
    Err(err) => ...
  };
}
```

Reply:
Resolved by #14587